metadata-ingestion/examples/ml/mlflow_dh_client_sample_ver2.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+
+from datahub.ingestion.graph.client import DataHubGraph
+from datahub.ingestion.graph.config import DatahubClientConfig
+
+from entities.experiment import Experiment
+from entities.model_group import Model_Group
+from entities.dataset import DataSetModel
+
+if __name__ == "__main__":
+    # Example usage
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", required=True, help="DataHub access token")
+    args = parser.parse_args()
+
+    client = DataHubGraph(
+        DatahubClientConfig(
+            server="http://localhost:8080",
+            token=args.token,
+            extra_headers={"Authorization": f"Bearer {args.token}"},
+        )
+    )
+
+    # Create Experiment
+    experiment = Experiment(experiment_id="airline_forecast_experiment",
+                            name="Airline Forecast Experiment",
+                            description="Experiment for forecasting airline passengers",
+                            platform="mlflow",
+                            custom_properties={"experiment_type": "forecasting"},
+                            client=client)
+
+    # Create Run from Experiments
+    run = experiment.create_run(
+        run_id="simple_training_run",
+        name="Simple Training Run")
+
+    # Create Model Group
+    model_group = Model_Group(
+        group_id="airline_forecast_models_group",
+        name="Airline Forecast Models Group",
+        description="Group of models for airline passenger forecasting",
+        time=1628580000000
+    )
+
+    # Create Model from ModelGroup
+    model = model_group.create_model(
```

Comment:
notes from discussion -- which direction are we going to follow? 

1. Only allows creating model with model group
* `model_group.create_model()`

2. Allows creating model without model and adding existing model with it 
* `client.create_model()` without model group
* `model_group.add_model()` that adds existing model to model group 


IMO for now i think it makes sense to do 1 and add 2 as a follow-up